### ISSUE TITLE
Unknown or uninitialised column: 'CHROM'

### ISSUE THREAD

**Issue by biopig** [bmansfeld/QTLseqR#7]
when i'm useing the qtlseqr, and i have  a promble, i can find the mathod
my file is 'zhang'
```
CHROM	POS	REF	ALT	H.AD	H.DP	H.GQ	H.PL	L.AD	L.DP	L.GQ	L.PL
Chr1	1984	T	C	19,0	19	57	0,57,956	10,3	13	87	87,0,420
Chr1	1994	C	T	19,1	20	34	0,34,532	10,4	14	61	61,0,297
Chr1	1995	G	A	21,0	21	63	0,63,1046	11,3	14	84	84,0,436
Chr1	2025	G	A	20,7	27	99	128,0,515	17,4	21	75	75,0,515
Chr1	2031	C	T	31,0	31	93	0,93,1355	19,5	24	76	76,0,667
Chr1	16386	C	A	3,2	5	75	75,0,140	6,0	6	18	0,18,250
Chr1	16402	TA	T	5,0	5	15	0,15,185	3,2	5	49	49,0,106
Chr1	16417	G	C	5,2	7	69	69,0,187	7,0	7	21	0,21,277
Chr1	17148	T	A	16,6	22	68	68,0,526	18,0	18	60	0,60,663
```

```
high<-"H"
low<-"L"
df<-importFromGATK(file="zhang",highBulk=high,lowBulk=low,chromList="Chr1")
Parsed with column specification:
cols(
  CHROM = col_integer(),
  POS = col_integer(),
  REF = col_character(),
  ALT = col_character(),
  H.AD = col_character(),
  H.DP = col_integer(),
  H.GQ = col_integer(),
  H.PL = col_character(),
  L.AD = col_character(),
  L.DP = col_integer(),
  L.GQ = col_integer(),
  L.PL = col_character()
)
Removing the following chromosomes: 
Warning messages:
1: Unknown or uninitialised column: 'CHROM'. 
2: Unknown or uninitialised column: 'CHROM'. 
3: Unknown or uninitialised column: 'CHROM'. 
4: Length of logical index must be 1 or 99, not 0 
5: Unknown or uninitialised column: 'CHROM'. 
6: Unknown or uninitialised column: 'CHROM'. 
```

**Comment by biopig** [comment]
@bmansfeld

**Comment by bmansfeld** [comment]
Hi,
For some reason the readr::read_tsv function is interpreting you CHROM column as an integer:
See:
```
Parsed with column specification:
cols(
  CHROM = col_integer(),
```
It is hard for me to understand why without your full data file. If you would like, and feel comfortable sending me the actual file you can: mansfeld (at) msu.edu 
Ben

**Comment by biopig** [comment]
what is you email

**Comment by biopig** [comment]
@bmansfeld i have send my file to you email

**Comment by bmansfeld** [comment]
Hi,
I did not receive the file. Perhaps it was too large to send. I recommend uploading it to a Dropbox or drive and sharing a link we me.
Thanks

**Comment by bmansfeld** [comment]
Hello, 
Thanks for sending your file. I have figured out the problem.
Since your bulks were named "H" and "L" and there is a column renaming step in the import function,
the H in CHROM was being renamed to HIGH.
I have fixed this in the current v 0.6.5.
Thanks for helping me find this bug.
Ben
